### ISSUE TITLE
Travis is unhappy about pandas

### ISSUE THREAD

**Issue by twiecki** [quantopian/pyfolio#387]
```
======================================================================
ERROR: Failure: ImportError (cannot import name PandasError)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/quantopian/pyfolio/pyfolio/__init__.py", line 3, in <module>
    from . import utils
  File "/home/travis/build/quantopian/pyfolio/pyfolio/utils.py", line 27, in <module>
    from pandas_datareader import data as web
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/pandas_datareader/data.py", line 7, in <module>
    from pandas_datareader.google.daily import GoogleDailyReader
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/pandas_datareader/google/daily.py", line 1, in <module>
    from pandas_datareader.base import _DailyBaseReader
  File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/pandas_datareader/base.py", line 11, in <module>
    from pandas.core.common import PandasError, is_number
ImportError: cannot import name PandasError
----------------------------------------------------------------------
```

**Comment by twiecki** [comment]
There are quite a few other errors. @gusgordon do you have time to take a look?

**Comment by gusgordon** [comment]
Not this week (I'm away for a few days) but probably next week.

**Comment by twiecki** [comment]
@gusgordon Sorry for the ping, that's more than fine.

**Comment by twiecki** [comment]
Fixed by #395 .
